pkg/types/k6cli.go

```diff
@@ -14,12 +14,10 @@ type CLI struct {
 
 func ParseCLI(arguments string) *CLI {
 	lastArgV := func(start int, args []string) (end int) {
-		var nextArg bool
 		end = start
-		for !nextArg && end < len(args) {
+		for end < len(args) {
```

Comment:
I don't remember why it was written with `nextArg` :sweat_smile: But the change seems to be correct and the unit test is OK... For my understanding, what was the linter that made you add this change?

Reply:
```
pkg/types/k6cli.go:22:5: ineffectual assignment to nextArg (ineffassign)
				nextArg = true
				^
```

Reply:
Got it, thanks!